RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -2817,6 +2829,7 @@
 				SKIP_INSTALL = YES;
 				SUPPORTED_PLATFORMS = "macosx iphonesimulator iphoneos watchsimulator watchos appletvsimulator appletvos";
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
+				SWIFT_TREAT_WARNINGS_AS_ERRORS = NO;
```

Comment:
💡 I know you did this temporarily, just leaving a note here so we remember to revert it